src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const check = Object.values(robot);
+
+  for (let i = 0; i < check.length; i++) {
+    if (check.indexOf(check[i]) !== i) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
```

Comment:
you can simplify the code and use one `for` loop. just add the check for second loop and return `null` if needed